### PR TITLE
add print in function _check_manifest_image

### PR DESCRIPTION
**Pull request by kaadevelop**

### CODE REVIEW

File: pylint_odoo/checkers/itp_checks.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def _is_string_instance(obj):
         settings.DESC_DFLT
     ),
     'E%d91' % settings.BASE_OMODULE_ID: (
-        'Module\'s image is absent or it\'s not in images/ folder. Check attribute "images" in module\'s __manifest__.py',
+        'Module\'s image is absent or it\'s not in images/ folder or image has invalid name. Check attribute "images" in module\'s __manifest__.py',
```
Comment (yelizariev) [inline pylint_odoo/checkers/itp_checks.py]: :-1:

File: pylint_odoo/checkers/itp_checks.py [hunk 2]
```diff
@@ -141,7 +141,8 @@ def _check_manifest_image(self):
             images = manifest_dict.get('images', [])
             if images != []:
                 for image in images:
-                    if image.startswith("images/"):
+                    if "images/" in image:
```
Comment (yelizariev) [inline pylint_odoo/checkers/itp_checks.py]: Why did you replaced it?

File: pylint_odoo/checkers/itp_checks.py [hunk 3]
```diff
@@ -141,7 +141,8 @@ def _check_manifest_image(self):
...
+                        print('Path to image from __manifest.py__ is %s' % (image))
```
Comment (yelizariev) [inline pylint_odoo/checkers/itp_checks.py]: print it only when file is absent, please

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: pylint_odoo/checkers/itp_checks.py (hunk 1 above)
Comment (kaadevelop) [reply pylint_odoo/checkers/itp_checks.py]: if will be invalid image's name than pylint speak about this too

File: pylint_odoo/checkers/itp_checks.py (hunk 2 above)
Comment (kaadevelop) [reply pylint_odoo/checkers/itp_checks.py]: I think it's correct too and it's short

Comment (yelizariev) [reply pylint_odoo/checkers/itp_checks.py]: It's not the same. It allows file name static/images/1.png, which is not correct

File: pylint_odoo/checkers/itp_checks.py (hunk 1 above)
Comment (yelizariev) [reply pylint_odoo/checkers/itp_checks.py]: The only invalid name is absent file

File: pylint_odoo/checkers/itp_checks.py [hunk 4]
```diff
@@ -142,7 +142,9 @@ def _check_manifest_image(self):
             if images != []:
                 for image in images:
                     if image.startswith("images/"):
-                        return os.path.isfile(os.path.join(self.module_path, image))
+                        if not os.path.isfile(os.path.join(self.module_path, image)):
+                            print('Path to image from __manifest.py__ is %s' % (image))
+                            return False
```
Comment (yelizariev) [inline pylint_odoo/checkers/itp_checks.py:149]: The method never returns True

**Pull request merged**